src/components/EditMovieModal/EditMovieModal.js

```diff
@@ -0,0 +1,164 @@
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+import CustomButton from "../CustomButton/CustomButton.js";
+import CustomInput from "../CustomInput/CustomInput.js";
+import CustomMultiselect from "../CustomMultiselect/CustomMultiselect.js";
+import "./style.scss";
+
+const fetchMovie = () => ({
+  title: "Some movie",
+  releaseDate: "2018-07-22",
+  movieUrl: "https://thumbs.dfs.ivi.ru/storage38/contents/a/b/f291cdd28726eb4d9f54e0b25a40b5.jpg",
+  genre: ["comedy", "crime"],
+  overview: "Movie overview",
+  runtime: "120 min",
+});
+
+const genreOptions = [
+  {
+    name: 'Drama',
+    value: 'drama',
+  }, {
+    name: 'Crime',
+    value: 'crime',
+  }, {
+    name: 'Comedy',
+    value: 'comedy',
+  }
+];
+
+class EditMovieModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movie: {
+        title: "",
+        releaseDate: "",
+        movieUrl: "",
+        genre: [],
+        overview: "",
+        runtime: "",
+      },
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
```

Comment:
Usually, we just use arrow functions to not bind methods

Reply:
It looks like this syntax is still in proposal state, but  itcan be handled with the additional babel plugin